Makefile

```diff
@@ -306,6 +307,7 @@ HUSKY          ?= $(LOCALBIN)/husky
 NILAWAY        ?= $(LOCALBIN)/nilaway
 GOVULNC        ?= $(LOCALBIN)/govulncheck
 MOCKGEN        ?= $(LOCALBIN)/mockgen
+GOWRAP         ?= $(LOCALBIN)/gowrap
```

Comment:
since this isn't supported by devbox can we use `CACHE_BIN` here instead of `LOCALBIN`?